src/index.js

```diff
@@ -49,6 +52,74 @@ const EmojiPicker = ({
     []
   );
 
+  useEffect(() => {
+    let unsubscribe = tinykeys(window, {
+      ArrowUp: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowDown: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowLeft: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowRight: keyEvent => handleKeyboard(keyEvent.key),
+    });
+    return () => {
+      unsubscribe();
+    };
+  }, []);
+
+  const handleKeyboard = key => {
+    const activeElementClass = document.activeElement.getAttribute('class');
+
+    if (activeElementClass && activeElementClass.startsWith('icn-')) {
+      handleCategoriesKeyboard(key);
+    } else if (activeElementClass === 'emoji-search') {
+      handSearchKeyboard(key);
+    } else {
+      handleEmojiesKeyboard(key);
+    }
+  };
+
+  const handleCategoriesKeyboard = key => {
+    switch (key) {
+      case ArrowRight:
+        document.activeElement.nextElementSibling.focus();
+        break;
+      case ArrowLeft:
+        document.activeElement.previousElementSibling.focus();
+        break;
+      case ArrowDown:
+        const searchInput = document.getElementsByClassName('emoji-search')[0];
+        searchInput.focus();
+        break;
+    }
+  };
+
+  const handSearchKeyboard = key => {
+    if (key === ArrowUp) {
+      const emojiCategories = document.getElementsByClassName(
+        'emoji-categories'
+      )[0];
+      emojiCategories.firstChild.focus();
+    }
+    if (key === ArrowDown) {
+      const firstEmoji = document.getElementsByClassName('emoji')[0];
+      firstEmoji.firstChild.focus();
+    }
+  };
+
+  const handleEmojiesKeyboard = key => {
+    if (key === ArrowRight) {
+      document.activeElement.parentElement.nextSibling.firstChild.focus();
+    }
+    if (key === ArrowLeft) {
+      const prevSibling = document.activeElement.parentElement.previousSibling;
+
+      if (prevSibling) {
+        prevSibling.firstChild.focus();
+      } else {
+        const searchInput = document.getElementsByClassName('emoji-search')[0];
+        searchInput.focus();
+      }
```

Comment:
The previous comments apply here as well, plus, there's a bug 😛

If you're in the first emoji category, ArrowLeft takes you back to search as it should, but if you're in any other emoji category, ArrowLeft doesn't take you to the previous category, but to the search as well. Also, if the search bar is not present, it has the same bug from above.

Might be worth taking this whole part out of the PR and taking it again in another PR so we have better visibility on it.